Cargo.toml

```diff
@@ -1,25 +1,22 @@
-[package]
-name = "kurbo"
+[workspace]
+resolver = "2"
+members = ["kurbo"]
+
+[workspace.package]
+# Kurbo version, also used by other packages which want to mimic Kurbo's version.
+#
+# NOTE: When bumping this, remember to also bump the aforementioned other packages'
+#       version in the dependencies section at the bottom of this file.
 version = "0.11.1"
-authors = ["Raph Levien <raph.levien@gmail.com>"]
```

Comment:
I see that this is not added back; that's fine and is probably the right thing to do, but it's worth calling it out explicitly.

Reply:
Putting it back in and we can do it explicitly.